### PR TITLE
Finished UT of login command

### PR DESCRIPTION
**Pull request by zhanggbj**
Hi @maximilien ,

Finished rest UT of login command. Thanks.

### CODE REVIEW

File: clients/fakes/bmp_client_fake.go [hunk 1]
```diff
@@ -9,7 +9,7 @@ type FakeBmpClient struct {
 	Password string
 	Url      string
 
-	ConfigPathString string
+	configPath string
```
Comment (maximilien) [inline clients/fakes/bmp_client_fake.go]: Wrong. In takes variables should always be public (start with capital)

File: clients/fakes/bmp_client_fake.go [hunk 2]
```diff
@@ -41,15 +41,15 @@ type FakeBmpClient struct {
...
 func NewFakeBmpClient(username, password, url string, configPath string) *FakeBmpClient {
 	return &FakeBmpClient{
-		Username:         username,
-		Password:         password,
-		Url:              url,
-		ConfigPathString: configPath,
+		Username:   username,
+		Password:   password,
+		Url:        url,
+		configPath: configPath,
```
Comment (maximilien) [inline clients/fakes/bmp_client_fake.go]: Make public. Always in fakes

**Comment by maximilien** [comment]
Please address my comment and I'll merge. Thx

**Comment by zhanggbj** [comment]
Changed back to ConfigPathString, otherwise the name will conflict with the ConfigPath(). Thank you.

**Comment by maximilien** [comment]
cool, make sure to squash commits next time as I will no longer merge meaningless commits like `"fix typo"`

**Pull request merged**